pycaret/time_series.py

```diff
@@ -968,7 +987,7 @@ def plot_model(
     groups: Optional[Union[str, Any]] = None,
     use_train_data: bool = False,
     verbose: bool = True,
-    display_format: Optional[str] = None
+    display_format: Optional[str] = None,
```

Comment:
Hi @TremaMiguel , In general, I see a lot of formatting changes unrelated to functionality. Are you using an autoformatter that is adding these?

Reply:
@Yard1, In my code, I set a limit of 120 chars for the line. What are we using for PyCaret (80 or 120)?

Reply:
ok, agree on 120 when running black, I had the default to 80 lines.